pkg/minikube/out/out_test.go

```diff
@@ -114,3 +114,7 @@ func TestErr(t *testing.T) {
 		t.Errorf("Err() = %q, want %q", got, want)
 	}
 }
+
+func TestInfof(t *testing.T) {
```

Comment:
empty test

Reply:
thanks, removed. this is tested in `TestOutT` test